core-libs/setup/ssr/error-handlers/ssr-error-handler.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ErrorHandler, Injectable, inject } from '@angular/core';
+import { LoggerService } from '@spartacus/core';
+import { loggerEnabled } from '../logger';
+
+export const ssrErrorHandlerFactory = () => {
+  const isLoggerEnabled = inject(loggerEnabled);
+
+  return isLoggerEnabled ? new SsrErrorHandler() : new ErrorHandler();
```

Comment:
[QUESTION] 

Why do we do `new SsrErrorHandler` instead of `inject(SsrErrorhandler)`?

Reply:
Thanks, good catch. It was made by mistake.